### PR TITLE
Update --no-logs

### LINKED ISSUE #231: [Refactor] LRO pollers
Should be applicable to `run` as well and agnostic to `build`

### PR DESCRIPTION
**Pull request by ehotinger**
Switch from deserializing into `Build` into the new `Run` model.

Fixes #231

### CODE REVIEW

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py [hunk 1]
```diff
@@ -325,7 +325,7 @@ def acr_build_task_run(cmd,
     logger.warning("Waiting for build agent...")
```
Comment (mnltejaswini) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py:325]: do you want rename run agent?

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py [hunk 2]
```diff
@@ -325,7 +325,7 @@ def acr_build_task_run(cmd,
...
 
     if no_logs:
-        return get_build_with_polling(client, build_id, registry_name, resource_group_name)
+        return get_run_with_polling(client, build_id, registry_name, resource_group_name)
```
Comment (mnltejaswini) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py:328]: run_id or id?

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py [hunk 3]
```diff
@@ -103,8 +105,7 @@ def acr_run(cmd,
     logger.warning("Queued a run with ID: %s", id)
     logger.warning("Waiting for agent...")
```
Comment (mnltejaswini) [inline src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/run.py:106]: to be consistent with other, run agent?

File: src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py (hunk 2 above)
Comment (ehotinger) [reply src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/build_task.py:328]: @mnltejaswini I'll make a big item for this, thanks. https://github.com/AzureCR/azure-cli/issues/243

**Pull request merged**